myapi/serializers.py

```diff
@@ -1,19 +1,28 @@
 from rest_framework import serializers
 
 from .models import Hero
-from .models import Restaurant, Taco
+from .models import Restaurant, Taco, Review
 
 class HeroSerializer(serializers.HyperlinkedModelSerializer):
     class Meta:
         model = Hero
         fields = ('name', 'alias')
 
-class TacoSerializer(serializers.ModelSerializer):
+class ReviewSerializer(serializers.ModelSerializer):
     class Meta:
-        model = Taco
+        model = Review
         # this might need to be changed
         fields = "__all__"
 
+class TacoSerializer(serializers.ModelSerializer):
+    reviews = ReviewSerializer(many=True, read_only=True)
+    average_rating = serializers.ReadOnlyField()
+
+    class Meta:
+        model = Taco
+        # the field reviews must be named exactly the same as the related name
+        fields = ('id', 'type', 'restaurant_id', 'average_rating', 'reviews')
+
```

Comment:
Front end was cool with removing vegan bool in the taco fields? (im cool with it)

Reply:
Yeah, I asked about it and they decided to just make vegan a taco type rather than its own field